packages/story-editor/src/app/story/useStoryReducer/test/copySelectedElement.js

```diff
@@ -71,6 +96,51 @@ describe('copySelectedElement', () => {
     });
   });
 
+  it("copies an element's styles if no animations", () => {
```

Comment:
```suggestion
  it("copies an elements styles if no animations", () => {
```

Reply:
Isn't that a possessive apostrophe? Seems correct to me, though I'm not a native speaker

Reply:
Changed both instances to "the styles of an element" in de57669eac.